app/assets/stylesheets/mo/_utilities.scss

```diff
@@ -609,6 +609,10 @@
   text-indent: 0;
 }
 
+.violation-highlight {
+  background-color: orange;
+}
+
```

Comment:
@nimmolo: How can I style the Constraints Violation button so that it works with:
- any User theme? 
- screen widths <= 766 px?
What I've done here isn't satisfactory.

Reply:
Thinking about this. Need a bit of time to delve into those CSS variables

Reply:
@nimmolo: I've re-thought this, instead of changing the background, I'll use a colored border.
- It should be easier to find a non-enauseating color;
- It should be easier to find a color that works with all themes; 
- Perhaps it will solve the problem of the theme not working at small screen widths.

Reply:
That sounds good... i think you can use "border-danger" and it should contrast in all themes, but i'm not sure.

Reply:
Thanks for the suggestion. I'll have to dig into the styling some more.
"border-danger" doesn't seem to work on its own.
I finally ended up changing the text color.
Resolving conversation.